src/test/java/com/uber/cadence/workflow/WorkflowMigrationTest.java

```diff
@@ -183,18 +206,18 @@ public void whenUseDockerService_continueAsNewWorkflowMigration() {
           .execute(0);
     } catch (CancellationException e) {
       try {
-        describeWorkflowExecution(workflowClientNew, workflowID);
+        getWorkflowHistory(workflowClientNew, workflowID);
       } catch (Exception eDesc) {
         fail("fail to describe workflow execution in new domain: " + eDesc);
       }
     }
   }
 
-  private DescribeWorkflowExecutionResponse describeWorkflowExecution(
+  private GetWorkflowExecutionHistoryResponse getWorkflowHistory(
```

Comment:
Nit: Why change the type? Does this cover a different code path?

Reply:
describe method is not implemented in test service